nats-concepts/jetstream/object-store/obj_store.md

```diff
@@ -1,5 +1,11 @@
 # Object Store
 
+{% hint style="warning" %}
+If your objects fit within the JetStream [maximum message size](/running-a-nats-service/configuration#limits) (default 1MB, not recommended above 8MB), you should use the [Key/Value Store](/nats-concepts/jetstream/key-value-store/) instead.
+
+KV store is inherently simpler and more reliable for smaller data, providing strong consistency and atomic operations without the complexity of chunking.
```

Comment:
"more reliable" is quite ambigous. Same with "strong consistency" - One could argue about those.

I would simplify and just mention that for small values, consider using KV before switching to Object Store.